tests/test_main.py

```diff
@@ -51,6 +54,17 @@ def configuration(self):
             "population.yaml",
         ]
         project_config_dict = load_yamls(project_configs)
+        hapi_countries = ["AFG", "BFA", "MLI", "NGA", "TCD", "YEM"]
+        countries_to_remove = [
+            country
+            for country in project_config_dict["HAPI_countries"]
+            if country not in hapi_countries
+        ]
+        project_config_dict["HAPI_countries"] = hapi_countries
+        project_config_dict = subset_scraper_countries(
+            project_config_dict,
+            countries_to_remove,
+        )
```

Comment:
@mcarans I created a function "subset_scraper_countries" to remove scrapers from the configs as a workaround to changing hdx-python-scraper.  It feels like adding this functionality to hdx-python-scraper isn't worth the number of times we would use it, but if you think it would be useful I'm happy to look into it!